packages/venia-ui/lib/components/ProductOptions/tile.css

```diff
@@ -3,8 +3,6 @@
     border: 1px solid rgb(var(--venia-text));
     border-radius: 2px;
     height: 3rem;
```

Comment:
Make sure to fix the media query that is busted with the 2rem height :o

Reply:
Also fixed.